drivers/include/edbg_eui.h

```diff
@@ -30,6 +30,9 @@ extern "C" {
 /**
  * @brief   Get the unique EUI64 address from a Atmel EDBG debugger
  *
+ * @note    The EDBG firmware takes a while to start up, so on cold
+ *          boot this function can return error when called to soon.
```

Comment:
```suggestion
 * @note    The EDBG firmware may take a while to start up, so on cold
 *          boot this function can return error when called to soon.
```

Or is it guaranteed that this is always the case?

Reply:
It's a very deterministic system.
I always measured the same amount of retries (1124) before I would get an address (I did measure with `tests/driver_edbg_eui`, but since EUID assignment happens before main is called, the number is a bit higher here.
I chose 10000 to be on the safe side should we get there faster in boot due to code restructuring.

Reply:
With "in general" I meant: "for all boards" ;-).

Reply:
Ok, I don't know if other boards have that issue too.
The `same54-xpro` replaced the EUI in edbg with an actual AT24MAC chip that doesn't have this issue.

I can make it a bit more vague :wink: